airflow/www/app.py

```diff
@@ -151,15 +152,18 @@ def integrate_plugins():
 
                 for v in flask_appbuilder_views:
                     log.debug("Adding view %s", v["name"])
-                    appbuilder.add_view(v["view"],
-                                        v["name"],
-                                        category=v["category"])
+                    if 'view' in v:
+                        v['baseview'] = v.pop('view')
+                        warnings.warn(
+                            "AirflowPlugin.flask_appbuilder_views[] dictionaries should use 'baseview' instead of 'view. "
+                            "Using 'view' has been deprecated and will be removed in future versions.",
+                            DeprecationWarning,
+                        )
+
+                    appbuilder.add_view(**v)
```

Comment:
baseview, name is not kwargs in https://flask-appbuilder.readthedocs.io/en/latest/api.html#flask_appbuilder.base.AppBuilder.add_view. Should we still change it to ```
                    appbuilder.add_view(v["baseview"],
                                        v["name"],
                                        category=v["category"])
```?

Reply:
i’m not sure I follow the benefit here? The intention of using kwargs is so we don’t have to worry about the underlying method interface. In code that a plugin author would never see, why would we lock down positional arguments, when we could drop all the arguments straight through to fab with kwargs?

Reply:
When you say kwargs, do you mean optional arguments?

Reply:
makes sense to me. We are changing view to basename, with a warning to make upgrading easier.